src/api/controllers/guilds.ts

```diff
@@ -112,11 +113,13 @@ export async function handleInternalGuildEmojisUpdate(data: DiscordPayload) {
   if (!guild) return;
 
   const cachedEmojis = guild.emojis;
-  guild.emojis = payload.emojis;
+  guild.emojis = new Collection(
```

Comment:
have u tested this, upon quick review i dont think this will work

Reply:
Tested what?